### PR TITLE
Community invite test + small fixes

### PR DESCRIPTION
**Pull request by jrainville**
First commit: fixes a small issue where if you invited someone, but that 1-1 chat wasn't active, you didn't see that chat or message.
Second commit: fix the login issue with using a data dir (thanks @endulab )
Third commit: Adds a test that invites a mutual contact to a community and verifies that we sent the message (does not rely on mailserver)

Edit: I renamed the `group_chat` DB fixture to `mutual_contact`, since I use it to invite the user.

### CODE REVIEW

**Comment by jrainville** [comment]
@stefandunca and @Khushboo-dev-cpp gentle reminder to review this

File: test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py [hunk 1]
```diff
@@ -122,3 +122,7 @@ def step(context, message_index):
 def step(context, amount):
     _statusCommunityScreen.check_pin_count(amount)
 
+@When("the admin invites the user named |any| to the community with message |any|")
+def step(context, user_name, message):
+    _statusCommunityScreen.invite_user_to_community(user_name, message)
```
Comment (stefandunca) [inline test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py:127]: I'm thinking that adding type hints and casting from gherkin input into types here is worth the small effort of sanitizing data input

**Review by stefandunca — APPROVED** [review]
Great job

I also verified it on mac.

File: ui/app/AppLayouts/Onboarding/views/LoginView.qml [hunk 2]
```diff
@@ -202,6 +202,7 @@ Item {
 
             StatusBaseText {
                 id: usernameText
+                objectName: "currentUserNameLabel"
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Onboarding/views/LoginView.qml:205]: maybe loginView_currentUsername?

**Comment by jrainville** [comment]
I rebased the PR. Let's wait for the CI build. If works, I'll merge. I also fixed the group chat test

**Pull request merged**